walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -12,12 +12,243 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/**
```

Comment:
nit: This should be right above the line "public final class FindMeetingQuery {"

Reply:
Thanks for pointing it out Mark!